src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1852,6 +1855,16 @@ private AuthenticationResult getTokenWithRefreshToken(final IWindowComponent act
                 result.setRefreshToken(refreshItem.mRefreshToken);
             }
         } catch (IOException | AuthenticationException exc) {
+            Class causeClass = exc.getCause() != null ? exc.getCause().getClass() : exc.getClass();
```

Comment:
And by that I mean the whole block of code that produces the analytics log entry.